numpy/core/numeric.py

```diff
@@ -384,12 +384,12 @@ def full_like(a, fill_value, dtype=None, order='K', subok=True, shape=None):
     return res
 
 
-def _count_nonzero_dispatcher(a, axis=None):
+def _count_nonzero_dispatcher(a, axis=None, *, keepdims=False):
```

Comment:
@shoyer shouldn't this be always `None` for the dispatching signature, did it actually matter?

Reply:
I think it was just convention to avoid defaults getting out of sync. Still worthwhile though.

Reply:
OK, I changed the `keepdims` default in the dispatcher function signature to None.

Reply:
Hmm, it looks like our import-time check to verify default values of dispatcher arguments may be broken on keyword only argument.